payjoin-cli/README.md

```diff
@@ -1,111 +1,187 @@
 
-While this code and design has had significant testing, it is still alpha-quality experimental software. Use at your own risk.
 Independent audit is welcome.
 
-## Install `payjoin-cli`
+## Quick Start
+
+Here's a minimal payjoin example using `payjoin-cli` with the `v2` feature connected to `bitcoind` on [regtest](https://developer.bitcoin.org/examples/testing.html#regtest-mode). This example uses [`nigiri`](https://github.com/vulpemventures/nigiri) to setup a regtest environment. 
+
+Payjoin `v2` allows for transactions to be completed asynchronously. Thus the sender and receiver do not need to be online at the same time to payjoin. To make this work, we need a payjoin directory server and OHTTP relay. Learn more about how `v2` works [here](https://payjoin.org/docs/how-it-works/payjoin-v2-bip-77).
+
+To get started, install `nigiri`. Payjoin requires the sender and receiver each to have spendable [UTXOs](https://www.unchained.com/blog/what-is-a-utxo-bitcoin), so we'll create two wallets and fund each with spendable UTXOs.
 
-```console
-cargo install payjoin-cli --version $VERSION
+```sh
+cargo install nigiri
+
+nigiri rpc createwallet "sender"
+nigiri rpc createwallet "receiver"
+
+# We need 101 blocks for the UTXOs to be spendable due to the coinbase maturity requirement.
+nigiri rpc generatetoaddress $(nigiri rpc getnewaddress "sender") 101
+nigiri rpc generatetoaddress $(nigiri rpc getnewaddress "receiver") 101
 ```
 
-where `$VERSION` is the [latest version](https://crates.io/crates/payjoin-cli) of the `payjoin-cli` you wish to install.
+Great! Our wallets are setup, now let's do an async payjoin.
 
-Get a list of commands and options:
+### Install `payjoin-cli`
 
-```console
-payjoin-cli --help
+```sh
+cargo install payjoin-cli --version $VERSION --features v2
 ```
 
-Either pass config options from cli, or manually edit a `config.toml` file within directory you run `payjoin-cli` from.
-Configure it like so:
+where `$VERSION` is the [latest version](https://crates.io/crates/payjoin-cli).
+
+Next, create a directory for the sender & receiver and create a `config.toml` file for each:
+
+```sh
+mkdir sender receiver
+touch sender/config.toml receiver/config.toml
+```
+
+Edit the `config.toml` files. Note that the `v2` feature requires a payjoin directory server and OHTTP relay. Learn more about them [here](https://payjoin.org/docs/how-it-works/payjoin-v2-bip-77).
 
 ```toml
-# config.toml
-bitcoind_cookie = "/tmp/regtest1/bitcoind/regtest/.cookie"
-# specify your wallet via rpchost connection string
-bitcoind_rpchost = "http://localhost:18443/wallet/boom"
- ```
+# sender/config.toml
 
-Your configuration details will vary, but you may use this as a template.
+# Nigiri uses the following RPC credentials
+bitcoind_rpcuser = "admin1"
+bitcoind_rpcpassword = "123"
+bitcoind_rpchost = "http://localhost:18443/wallet/sender"
```

Comment:
the new config in [the example](https://github.com/payjoin/rust-payjoin/blob/master/payjoin-cli/example.config.toml) shows how this syntax won't work any longer. It's separated into toml subsections